_config/config.yml

```diff
@@ -1,12 +1,16 @@
 ---
 Name: workable
 ---
-Workable:
-  cache_expiry: 3600
-Injector:
-  Workable:    
+SilverStripe\Core\Injector\Injector:
+  Psr\SimpleCache\CacheInterface.workable:
+    factory: SilverStripe\Core\Cache\CacheFactory
+    constructor:
+      namespace: 'workable'
+      defaultLifetime: 3600
+  SilverStripe\Workable\Workable:
     constructor:
       0: %$WorkableRestfulService
+      1: %$Psr\SimpleCache\CacheInterface.workable
   WorkableRestfulService:
```

Comment:
We don't really do these arbitrary names anymore. It should be the FQCN, or the FQCN of an interface, with a dot-syntax, e.g. `My\Interface.someImplementation`.

Reply:
Sorry, the wrong line is highlighted up there. Should be line 10.